{"url": "https://api.github.com/repos/tensorflow/tensorflow/issues/comments/395765546", "html_url": "https://github.com/tensorflow/tensorflow/issues/17289#issuecomment-395765546", "issue_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/17289", "id": 395765546, "node_id": "MDEyOklzc3VlQ29tbWVudDM5NTc2NTU0Ng==", "user": {"login": "saeed68gm", "id": 6844416, "node_id": "MDQ6VXNlcjY4NDQ0MTY=", "avatar_url": "https://avatars0.githubusercontent.com/u/6844416?v=4", "gravatar_id": "", "url": "https://api.github.com/users/saeed68gm", "html_url": "https://github.com/saeed68gm", "followers_url": "https://api.github.com/users/saeed68gm/followers", "following_url": "https://api.github.com/users/saeed68gm/following{/other_user}", "gists_url": "https://api.github.com/users/saeed68gm/gists{/gist_id}", "starred_url": "https://api.github.com/users/saeed68gm/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/saeed68gm/subscriptions", "organizations_url": "https://api.github.com/users/saeed68gm/orgs", "repos_url": "https://api.github.com/users/saeed68gm/repos", "events_url": "https://api.github.com/users/saeed68gm/events{/privacy}", "received_events_url": "https://api.github.com/users/saeed68gm/received_events", "type": "User", "site_admin": false}, "created_at": "2018-06-08T13:46:17Z", "updated_at": "2018-06-08T13:46:17Z", "author_association": "NONE", "body_html": "<p><a class=\"user-mention\" data-hovercard-type=\"user\" data-hovercard-url=\"/hovercards?user_id=15064790\" data-octo-click=\"hovercard-link-click\" data-octo-dimensions=\"link_type:self\" href=\"https://github.com/nixingyang\">@nixingyang</a> yeah, correct. I did that because libtensorflow_inference.so was failing to build on master so I had to revert. I will try a bit later and see if the issues have been fixed for that. Do we know the commit that fixed this? I may just cherry-pick that and try.</p>", "body_text": "@nixingyang yeah, correct. I did that because libtensorflow_inference.so was failing to build on master so I had to revert. I will try a bit later and see if the issues have been fixed for that. Do we know the commit that fixed this? I may just cherry-pick that and try.", "body": "@nixingyang yeah, correct. I did that because libtensorflow_inference.so was failing to build on master so I had to revert. I will try a bit later and see if the issues have been fixed for that. Do we know the commit that fixed this? I may just cherry-pick that and try."}